commands.go

```diff
@@ -299,24 +300,34 @@ func importBind(args importArgs) {
 				// no difference - leave it untouched
 				delete(existing, key)
 			} else {
-				// new record, add
-				change := route53.Change{
-					Action:            aws.String("CREATE"),
-					ResourceRecordSet: rrset,
+				// new record, add or upsert
+				if args.upsert {
+					change := route53.Change{
+						Action:            aws.String("UPSERT"),
+						ResourceRecordSet: rrset,
+					}
+					additions = append(additions, &change)
+				} else {
+					change := route53.Change{
+						Action:            aws.String("CREATE"),
```

Comment:
This one could just be changed to always UPSERT if you wanted it simpler, but I went in direction of not changing any existing behavior.